src/ripple/app/tx/tests/Offer.test.cpp

```diff
@@ -23,31 +23,69 @@
 #include <ripple/basics/Log.h>
 #include <ripple/test/jtx.h>
 #include <ripple/test/jtx/Account.h>
+#include <ripple/ledger/tests/PathSet.h>
 
 namespace ripple {
 namespace test {
 
-// An offer exists
-bool isOffer (jtx::Env const& env,
-    jtx::Account const& account,
-    STAmount const& takerPays,
-    STAmount const& takerGets)
-{
-    bool exists = false;
-    forEachItem (*env.open(), account,
-        [&](std::shared_ptr<SLE const> const& sle)
-        {
-            if (sle->getType () == ltOFFER &&
-                sle->getFieldAmount (sfTakerPays) == takerPays &&
-                sle->getFieldAmount (sfTakerGets) == takerGets)
-                exists = true;
-        });
-    return exists;
-}
-
 class Offer_test : public beast::unit_test::suite
 {
 public:
+    void testRmFundedOffer ()
```

Comment:
Thanks so much for the unit test - each test is a step forward making other tests easier.